invenio_administration/assets/semantic-ui/js/invenio_administration/search/components.js

```diff
@@ -0,0 +1,193 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import React, { useState } from "react";
+import {
+  Accordion,
+  Button,
+  Card,
+  Checkbox,
+  Label,
+  List,
+} from "semantic-ui-react";
+import PropTypes from "prop-types";
+
+export const ParentFacetValue = ({
+  bucket,
+  keyField,
+  isSelected,
+  childAggCmps,
+  onFilterClicked,
+}) => {
+  const [isActive, setIsActive] = useState(false);
+
+  return (
+    <Accordion className="rdm-multi-facet">
```

Comment:
how about fixing these components in invenio-search-ui so that we don't need to copy over the code all the time ? ping @zzacharo